apps/demos/testing/widgets/contextmenu/ContextMenuShare.test.js

```diff
@@ -0,0 +1,25 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+fixture('ContextMenu.Share')
```

Comment:
It's better to name the fixture according to the demo name that you are testing. You test ContextMenu component, the name of demo is Templates. So, the fixture name should be 'ContextMenu.Templates'